### DISCUSSION TITLE
Using GEOSX for Carbon Mineralization.

### DISCUSSION THREAD

**Discussion by ghost** [GEOS-DEV/GEOS#2295] [Q&A] [answered]
I want to use GEOSX to simulate carbon mineralization in igneous rock formations, but I don't know if that is possible. I looked in the documentation in vain. 

Is that possible?

Thank you very much for your time.
Btw happy valentine's day.

**Comment by joshua-white** [comment, marked as answer]
@Rosknub, we do not have geochemistry support for this at the moment.  While this type of physics is on the long term development plan, you will need to look elsewhere at the moment.

**Reply by ghost** [reply]
Hello @joshua-white, thank you for answering me. I am going to look elsewhere.
